cassandra-dac/src/main/java/org/sunbird/cassandraimpl/CassandraOperationImpl.java

```diff
@@ -565,4 +565,60 @@ private void logQueryElapseTime(String operation, long startTime) {
     ProjectLogger.log(
         mf.format(new Object[] {operation, startTime, stopTime, elapsedTime}), LoggerEnum.PERF_LOG);
   }
+
+  @Override
+  public Response getRecordsByIndexedProperty(
+      String keyspaceName, String tableName, String propertyName, Object propertyValue) {
+    long startTime = System.currentTimeMillis();
+    ProjectLogger.log(
+        "Cassandra Service getRecordsByIndexedProperty method started at ==" + startTime,
+        LoggerEnum.INFO);
+    Response response = new Response();
+    try {
+      Select selectQuery = QueryBuilder.select().all().from(keyspaceName, tableName);
+      selectQuery.where().and(QueryBuilder.eq(propertyName, propertyValue));
+      ResultSet results = connectionManager.getSession(keyspaceName).execute(selectQuery);
+      response = CassandraUtil.createResponse(results);
+    } catch (Exception e) {
+      ProjectLogger.log(Constants.EXCEPTION_MSG_FETCH + tableName + " : " + e.getMessage(), e);
+      throw new ProjectCommonException(
+          ResponseCode.SERVER_ERROR.getErrorCode(),
+          ResponseCode.SERVER_ERROR.getErrorMessage(),
+          ResponseCode.SERVER_ERROR.getResponseCode());
+    }
+    logQueryElapseTime("getRecordsByIndexedProperty", startTime);
+    return response;
+  }
+
+  @Override
+  public Response deleteRecord(
+      String keyspaceName, String tableName, Map<String, String> compositeKeyMap) {
+    long startTime = System.currentTimeMillis();
+    ProjectLogger.log(
+        "Cassandra Service deleteRecord by composite key method started at ==" + startTime,
```

Comment:
From:
"Cassandra Service deleteRecord by composite key method started at =="

To:
"CassandraOperationImpl: deleteRecord by composite key called at "